bindings/python/docs/source/changelog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Changes in Version 0.5.0
+------------------------
+- Support auto-discovery of schemas.
```

Comment:
Worth mentioning the schema param is now option on the various find/aggregate_*_all methods. Also should we mention that it works by inferring the schema from the _first_ document in the result set?

Reply:
Done